src/utils/CvoPointCloud.cpp

```diff
@@ -720,6 +724,9 @@ namespace cvo{
     //bool is_recording_depth_map = true;
     //static unsigned int depth_map_counter = 0;
     //positions_.resize(pre_depth_selected_ind.size());
+    num_classes_ = raw_image.num_classes();
+    num_geometric_types_ = 2;
+    feature_dimensions_ = raw_image.channels() + 2;
```

Comment:
(Same comment above) Just to make sure, if the new CvoPointCloud a template class with template argument num_class and feature_dim? If so we don't need to assign it to be raw_image.channels() + 2;